tests/test_resolver.py

```diff
@@ -127,7 +128,7 @@ def generate_expected_resolved_deps(
 )
 @settings(
     suppress_health_check=[HealthCheck.function_scoped_fixture],
-    deadline=5000,
+    deadline=None,
```

Comment:
Argh 😅 It's a shame that we cannot access the `install_deps` parameter here, and them simply do:

```python
    deadline=None if install_deps else 5000,
```

Reply:
True :(